treap.cabal

```
@@ -79,10 +79,13 @@ test-suite treap-test
                          Test.Rand.Cuts
                          Test.Rand.Query
                          Test.Rand.Update
+                         Test.Rand.Laws
 
   build-depends:       treap
                      , hspec
                      , hspec-core
+                     , hedgehog >= 0.6
```

It's always a good idea to add upper and lower bounds
```suggestion
                     , hedgehog ^>= 0.6
```